e2e/output_runner/output_runner_test.go

```diff
@@ -69,9 +71,10 @@ func checkNoSentinel(t *testing.T, sentinelFile *os.File, msg string) {
 func checkSentinel(t *testing.T, sentinelFile *os.File, msg string) {
 	t.Helper()
 
+	time.Sleep(500 * time.Millisecond)
 	sentinalFileName := sentinelFile.Name()
 	if _, err := os.Stat(sentinalFileName); os.IsNotExist(err) {
-		t.Errorf("Couldn't find sentinal. os.Stat(%q): %s\n%s\n", sentinalFileName, err, msg)
+			t.Errorf("Couldn't find sentinal. os.Stat(%q): %s\n%s\n", sentinalFileName, err, msg)
```

Comment:
This formatting looks a little odd to me. Please remember to run `gofmt` over the repo before committing 😄

Reply:
Yes, sorry! thanks